packages/manager/apps/pci-public-ip/package.json

```diff
@@ -11,14 +11,15 @@
     "start": "lerna exec --stream --scope='@ovh-ux/manager-pci-public-ip-app' --include-dependencies -- npm run build --if-present",
     "start:dev": "lerna exec --stream --scope='@ovh-ux/manager-pci-public-ip-app' --include-dependencies -- npm run dev --if-present",
     "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/manager-pci-public-ip-app' --include-dependencies -- npm run dev:watch --if-present",
-    "test": "vitest run"
+    "test": "vitest run",
+    "test:watch": "vitest watch"
   },
   "dependencies": {
     "@ovh-ux/manager-config": "^7.3.0",
     "@ovh-ux/manager-core-api": "^0.7.0",
+    "@ovh-ux/manager-react-core-application": "0.8.2",
```

Comment:
Why do you need this package ?

Reply:
just moved it not add it :) but yeah it's for having the hook `useFeatureAvailability` to check the status of a Feature Flipping